buzzmobile/process/gps_mapper/gps_mapper.py

```diff
@@ -0,0 +1,65 @@
+#!/usr/bin/env python
+
+import rospy
+import math
+import polyline as pl
+import interpolate
+from cv_bridge import CvBridge, CvBridgeError
+from sensor_msgs.msg import NavSatFix
+from sensor_msgs.msg import Image
+from std_msgs.msg import Float64
+from std_msgs.msg import String
+
+class Frames(object):
```

Comment:
Don't use class for this. Instead, use a dict. For an example, see:

https://github.com/gtagency/buzzmobile/blob/c308441c251bc14262ad8c780b68129bd7a7955b/buzzmobile/sense/maps_querier/maps_querier.py#L18

There are reasons for this. One being that the `bearing` you're setting as `None` in the `__init__` is not the same as the `bearing` you're checking when doing `hasattr()`. The other being that we believe this creates a memory leak. See #62